sympy/core/tests/test_sympify.py

```diff
@@ -511,6 +511,37 @@ def test_sympify_set():
     assert sympify({n}) == FiniteSet(n)
     assert sympify(set()) == EmptySet()
 
+def test_numpy():
+    try:
+        import numpy as np
```

Comment:
You should use sympy.external.import_module here (look how it's done in the lambdify tests).